leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -304,6 +316,17 @@ protected DtlsConnectorConfig.Builder createRootDtlsConnectorConfigBuilder(
         builder.setBootstrapAdditionalAttributes(cli.main.bsAdditionalAttributes);
         final LeshanClient client = builder.build();
 
+        if (cli.main.factoryBootstrap != null)
+            for (Map.Entry<LwM2mPath, String> entry : cli.main.factoryBootstrap.entrySet()) {
+                LwM2mPath lwM2mPath = entry.getKey();
+                LwM2mNodeTextDecoder lwM2mNodeTextDecoder = new LwM2mNodeTextDecoder();
+                LwM2mSingleResource decodedResource = lwM2mNodeTextDecoder.decode(entry.getValue().getBytes(),
+                        lwM2mPath, repository.getLwM2mModel(), LwM2mSingleResource.class);
```

Comment:
We could support `LwM2mResourceInstance` too.

Reply:
Maybe supporting `LwM2mSingleResource` only is enough in a first time, because there is not so much multi instance resource to set in leshan-client-demo :thinking: 

any opinion ?

Reply:
I think supporting only LwM2mSingleResource will be fine at the beginning for me too.